chat-ui/.dockerignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+public/build
+src/_proto
+user-function.desc
+docker-compose.yaml
```

Comment:
How is about moving `docker-compose.yaml` to the repository root and removing this line?
That would be consistent with other sample repos, fro example https://github.com/cloudstateio/samples-js-chat